src/test/kotlin/io/snyk/plugin/ui/toolwindow/SuggestionDescriptionPanelFromLSTest.kt

```diff
@@ -101,8 +98,8 @@ class SuggestionDescriptionPanelFromLSTest : BasePlatformTestCase() {
     fun `test createUI should show nothing if feature flag is enabled but JCEF is not`() {
         pluginSettings().isGlobalIgnoresFeatureEnabled = true
 
-        mockkStatic("io.snyk.plugin.ui.UIUtilsKt")
-        every { getJBCefBrowserIfSupported() } returns (null to "")
+        mockkStatic("io.snyk.plugin.ui.jcef.UtilsKt")
```

Comment:
Any chance we can use objects instead of static mocks? Static mocks are... not nice ;).